spec/Statements.tex

```diff
@@ -789,15 +789,22 @@ \section{The Use Statement}
 C1, C2, C3 have public module level definitions of the same symbol.
 
 An optional \sntx{limitation-clause} may be provided to limit the symbols made
-available by the use statement.  If an \chpl{except} list is provided, then all
+available by a use statement.  If an \chpl{except} list is provided, then all
 the visible but unlisted symbols in the module or enumerated type will be made
 available without prefix.  If an \chpl{only} list is provided, then just the
 listed visible symbols in the module or enumerated type will be made available
 without prefix.  All visible symbols not provided via these limited use
-statements are still accessible with the module or enumerated type name prefix.
+statements are still accessible by qualifying the access by the module or enumerated type name prefix.
 It is an error to provide a name in a \sntx{limitation-clause} that does not
 exist or is not visible in the respective module or enumerated type.
 
+If a type is specified in the \sntx{limitation-clause}, then the type's fields
+and methods are treated similarly to the type name.  These fields and methods
+cannot be specified in a \sntx{limitation-clause} on their own.
+
+% We need to figure out what to do about functions that return types which due
+% to the limitation-clause are not visible without prefix.
```

Comment:
Yeaaaaaaah.  That's been sitting on my white board for three+ years at this point . . .